pkgs/os-specific/linux/kernel/generic.nix

```diff
@@ -37,7 +40,7 @@
   modDirVersion ? null
 
 , # An attribute set whose attributes express the availability of
-  # certain features in this kernel.  E.g. `{iwlwifi = true;}'
+  # certain features in this kernel.  E.g. `{ia32Emulation = true;}'
```

Comment:
Seems unrelated?

Reply:
I'll move the tiny feature cleanup to a separate PR.